channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/reactions/TelephonyReactions.kt

```diff
@@ -108,6 +113,71 @@ class TelephonyReactions(private val bargeInDefaultProps: BargeInProperties) : J
         bargeIn = BargeInProperties(mode, trigger, noInterruptTimeMs)
     }
 
+    /**
+     * This method overrides the TTS configuration of the phone channel used for the current call.
+     *
+     * Example usage:
+     * ```
+     * state("tts") {
+     *    action {
+     *        reactions.telephony?.setTtsConfig(TtsConfigAimyvoice("Никита"))
+     *    }
+     * }
+     * ```
+     * The configuration is determined by the TtsProviderConfig subclass passed in as a parameter.
+     * The appropriate provider-specific configuration settings are extracted and assigned
+     * to the corresponding fields in the TtsConfig class.
+     * @param config TtsProviderConfig subclass containing the provider-specific configuration settings to use for TTS.
+     * */
+    fun <T : TtsProviderConfig> setTtsConfig(config: T) {
+        ttsConfig = TtsConfig(
+            type = executionContext.request.telephony?.ttsConfig?.type,
+            yandex = config as? TtsConfigYandex,
+            google = config as? TtsConfigGoogle,
+            mts = config as? TtsConfigMts,
+            zitech = config as? TtsConfigZitech,
+            azure = config as? TtsConfigAzure,
+            aimyvoice = config as? TtsConfigAimyvoice,
+            sber = config as? TtsConfigSber
+        )
+    }
+
+    /**
+     * This method overrides the ASR configuration of the phone channel used for the current call.
+     * example usage:
+     * ```
+     * state("asr") {
+     *    action {
+     *        reactions.telephony?.setAsrConfig(
+     *            AsrYandexConfig(
+     *                model = "general",
+     *                lang = "ru-RU",
+     *                numbersAsWords = true,
+     *                sensitivityReduction = true
+     *           )
+     *        )
+     *    }
+     * }
+     * ```
+     * The configuration is determined by the AsrProviderConfig subclass passed in as a parameter.
+     * The appropriate provider-specific configuration settings are extracted and assigned
+     * to the corresponding fields in the AsrConfig class.
+     * @param config AsrProviderConfig subclass containing the provider-specific configuration settings to use for ASR.
+     * */
+    fun <T : AsrProviderConfig> setAsrConfig(config: T) {
```

Comment:
Same as setTtsConfig. We use type from request but not from config parameter. This looks strange.